overview.tex

```diff
@@ -15,10 +15,15 @@ \chapter{System Overview} \label{overview}
 OpenOCD, which may include a hardware driver) to communicate with Debug
 Transport Hardware (eg.  Olimex USB-JTAG adapter).
 The Debug Transport Hardware connects the Debug Host to the Platform's Debug
-Transport Module (DTM).  The DTM provides access to the Debug Module (DM) using
-the Debug Module Interface (DMI).
+Transport Module (DTM).  The DTM provides access to one or more Debug Modules
+(DMs) using the Debug Module Interface (DMI).
 
-The DM allows the debugger to halt any hart in the platform. Abstract commands
+Each hart in the system is controlled by exactly one DM. Harts may be
+heterogeneous. There is no further limit on the hart-DM mapping, but usually
+all harts in a single core are controlled by the same DM. In most systems there
```

Comment:
why do we have to talk about cores at all? Does the RISC-V ISA spec talk about cores?

Reply:
I added this because on the mailing list and the call there was some confusion about whether there can only be one DM per core or not. I agree that the DM doesn't care at all whether harts are on the same core or not.